src/misc.ts

```diff
@@ -235,7 +235,7 @@ export function hasErrors(errorList: UserError[] | null): boolean {
 export function getMutationState(
   called: boolean,
   loading: boolean,
-  ...errorList: UserError[][]
+  ...errorList: any[][]
```

Comment:
Why any?

Reply:
Well, we only need info `how many` errors were returned, it's no use for us to know `what` errors

Reply:
What do we gain from turning this into `any`? What if we wanted to explicitly forbid using `any` tomorrow to get stronger type checking?